alert-common/src/main/java/com/synopsys/integration/alert/common/rest/model/AlertPagedModel.java

```diff
@@ -27,8 +27,10 @@
 import net.minidev.json.annotate.JsonIgnore;
 
 public class AlertPagedModel<M extends AlertSerializableModel> extends AlertSerializableModel {
-    public static final String DEFAULT_PAGE_NUMBER = "0";
-    public static final String DEFAULT_PAGE_SIZE = "10";
+    public static final Integer DEFAULT_PAGE_NUMBER = 0;
+    public static final Integer DEFAULT_PAGE_SIZE = 10;
+    public static final String DEFAULT_PAGE_NUMBER_STRING = "0";
+    public static final String DEFAULT_PAGE_SIZE_STRING = "10";
```

Comment:
Spring requires a String constant in `ReadPageController`. We want to use the same defaults, but as an Integer. Should we convert the String to an Integer at runtime? Do what I've done here? Something else?

Reply:
```java
    public static final Integer DEFAULT_PAGE_NUMBER = 0;
    public static final Integer DEFAULT_PAGE_SIZE = 10;
    public static final String DEFAULT_PAGE_NUMBER_STRING = DEFAULT_PAGE_NUMBER.toString();
    public static final String DEFAULT_PAGE_SIZE_STRING = DEFAULT_PAGE_SIZE.toString();
```
based on https://stackoverflow.com/a/654068/6921621

Reply:
Then it is no longer a constant. 
![Screen Shot 2020-11-10 at 3 17 12 PM](https://user-images.githubusercontent.com/15365059/98728785-ef666180-2367-11eb-937e-34698c2a0f28.png)

Reply:
Alright, in that case maybe move the string constants into that interface and leave them as literals. Leave the integers in the paged model.

Reply:
You could reverse it, leave the static Strings as literals and have the static integers based on those>
```
public static final Integer DEFAULT_PAGE_NUMBER = Integer.valueOf(ReadPageController. DEFAULT_PAGE_NUMBER);
public static final Integer DEFAULT_PAGE_SIZE =  Integer.valueOf(ReadPageController. DEFAULT_PAGE_SIZE);
```

Reply:
I was hesitant to do the String parsing at runtime, but I think that's better than declaring the constants twice.